users.go

```diff
@@ -396,6 +396,7 @@ type SSHKey struct {
 	Title     string     `json:"title"`
 	Key       string     `json:"key"`
 	CreatedAt *time.Time `json:"created_at"`
+	ExpiresAt *time.Time `json:"expires_at"`
```

Comment:
Looking at the docs, should both of these be of type `*ISOTime`?

Reply:
Would be effectively the same. Will change to match other response structs.